sdk/testcontainers-vault/src/vault.ts

```diff
@@ -21,20 +21,28 @@ export class Vault {
       bindMounts = []
     } = this.options;
 
+    let pvaultServerOrDev = ''
+    if (typeof (this.options.isPvaultServer) == 'undefined') {
+      pvaultServerOrDev = `piiano/pvault-dev:`
+    }
+    else {
+      pvaultServerOrDev = 'piiano/pvault-server:'
+    }
+
```

Comment:
Add `withDB` to the extraction of options above and set the default to `true`:
```diff
const {
  version = 'latest',
  env = {},
  license = process.env.PVAULT_SERVICE_LICENSE,
  port,
+ withDB = true,
  bindMounts = []
} = this.options;
```

Rename `pvaultServerOrDev` to `vaultImage` and set it with a simple ternary expression:
```suggestion
    const vaultImage = withDB ? 'piiano/pvault-dev' : 'piiano/pvault-server'
```